airflow/sensors/external_task_sensor.py

```diff
@@ -161,3 +164,49 @@ def poke(self, context, session=None):
 
         session.commit()
         return count == len(dttm_filter)
+
+
+class ExternalTaskMarker(DummyOperator):
+    """
+    Use this operator to indicate that a task on a different DAG depends on this task.
+    When this task is cleared with "Recursive" selected, Airflow will clear the task on
+    the other DAG and its downstream tasks recursively. Transitive dependencies is followed
+    until the recursion_depth is reached.
+    """
```

Comment:
```suggestion
        :param external_dag_id: The dag_id that contains the task you want to wait for
        :type external_dag_id: str
        :param external_task_id: The task_id that contains the task you want to wait for.
        :type external_task_id: str
        :param execution_date: The execution_date of the task that you want to wait for.
        :type execution_date: str or datetime.datetime
        :param recursion_depth: The maximum level of transitive dependencies allowed. Default is 10.
            This is mostly used for preventing cyclic dependencies. It is fine to increase
            this number if necessary. However, too many levels of transitive dependencies will make
            it slower to clear tasks in the web UI.
        """
```

Reply:
Thanks for pointing out. I made this change as suggested.